package.json

```diff
@@ -1,66 +1,79 @@
 {
   "private": true,
-  "license": "MIT",
+  "engines": {
+    "node": ">=6",
+    "npm": ">=3"
+  },
   "scripts": {
-    "install": "run-s -s install:*",
-    "install:mkdirp-bin": "lerna exec -- mkdirp ./node_modules/.bin/",
-    "install:bootstrap": "lerna bootstrap",
-    "install:symlink-bin": "lerna exec -- node ../../scripts/symlink-bin.js",
-    "clean": "lerna exec -- rm -rf lib/* && lerna clean --yes",
-    "lint": "lerna exec -- npm run lint -s",
-    "build": "lerna exec -- npm run build -s",
-    "test": "lerna exec -- npm run test -s",
-    "coverage": "lerna exec -- npm run coverage -s",
-    "codecov": "lerna exec -- npm run codecov -s",
-    "commit": "git-cz || true",
-    "check-updates": "run-p -s check-updates:*",
-    "check-updates:root": "ncu --error-level 2",
-    "check-updates:packages": "lerna exec -- ncu --error-level 2",
```

Comment:
It's not removed, the check part has been moved to the `check-updates` script. It's also something you won't do that often and we should use something like `greenkeeper` for this sort of stuff.

This is what you get now anyway, which I think is more then enough:

```bash
$ npm run check-updates

> @ check-updates /Users/emmenko/dev/src/nodejs
> ./scripts/check-updates

Using /Users/emmenko/dev/src/nodejs/package.json
⸨░░░░░░░░░░░░░░░░░░⸩ ⠧ :
 eslint-config-commercetools  4.0.2  →  5.0.0
 eslint-plugin-jsx-a11y       2.2.3  →  3.0.1
Dependencies not up-to-date
Lerna v2.0.0-beta.30
Independent Versioning Mode
Using /Users/emmenko/dev/src/nodejs/packages/commercetools-sdk-middleware-auth/package.json
Using /Users/emmenko/dev/src/nodejs/packages/commercetools-sdk/package.json
Using /Users/emmenko/dev/src/nodejs/packages/commercetools-sdk-middleware-http/package.json
⸨░░░░░░░░░░░░░░░░░░⸩ ⠴ :
All dependencies match the latest package versions :)

⸨░░░░░░░░░░░░░░░░░░⸩ ⠧ :
All dependencies match the latest package versions :)

⸨░░░░░░░░░░░░░░░░░░⸩ ⠦ :
All dependencies match the latest package versions :)

Waiting for 1 child process to exit. CTRL-C to exit immediately.
```

The upgrade part I don't find it necessary and can be dangerous if done "automatically" without thinking much what to upgrade.